include/OpenSpaceToolkit/Astrodynamics/Flight/Profile.hpp

```diff
@@ -59,6 +74,35 @@ class Profile
         Custom      ///< Custom pointing mode
     };
 
+    enum class Axis
+    {
+        X,
+        Y,
+        Z
+    };
+
+    enum class TargetType
+    {
+        GeocentricNadir,
+        GeodeticNadir,
+        Trajectory,
+        Sun,
+        Moon,
+        Velocity
+    };
+
+    struct Target
+    {
+        TargetType type;
+        Axis axis;
+        bool antiDirection = false;
+
+        Integer getSign() const
+        {
+            return antiDirection ? -1 : 1;
+        }
```

Comment:
I wonder if we shouldn't that struct "pure" and have a simple inline function defined where `getSign` is used.